src/test/java/com/serenitydojo/fruitmarket/TheCatalog.java

```diff
@@ -10,49 +11,43 @@
 
 public class TheCatalog {
 
-    @DisplayName("The catalog should return the current price of a fruit")
+    Catalog catalog = new Catalog();
+
+    @BeforeEach
+    public void setUpCatalog(){
+        catalog.setPrice("Apples", 4.00);
+        catalog.setPrice("Oranges", 5.50);
+        catalog.setPrice("Bananas", 6.00);
+        catalog.setPrice("Pears", 4.50);
+    }
+    @DisplayName("Should return the current price of a fruit")
```

Comment:
method says `shouldBeAbleToUpdateTheCurrentPriceOfAFruit` but the display name says something different